integrationTests/relayers/slowTests/common.go

```diff
@@ -53,8 +53,8 @@ func GenerateTestUSDCToken() framework.TestTokenParams {
 				MvxSCCallData:        createScCallData("callPayable", 50000000),
 			},
 		},
-		ESDTSafeExtraBalance:    big.NewInt(100),                                        // extra is just for the fees for the 2 transfers mvx->eth
-		EthTestAddrExtraBalance: big.NewInt(-5000 + 2500 - 50 - 7000 + 300 - 50 - 1000), // -(eth->mvx) + (mvx->eth) - fees
+		ESDTSafeExtraBalance:    big.NewInt(100),                  // extra is just for the fees for the 2 transfers mvx->eth
+		EthTestAddrExtraBalance: big.NewInt(2500 - 50 + 300 - 50), // -(eth->mvx) + (mvx->eth) - fees
```

Comment:
maybe refactor this to have here a map for all involved addresses (Alice, Bob, Charlie...)